### PR TITLE
13.0 [MIG] pos_order_remove_line

### PR DESCRIPTION
**Pull request by JSeguiN97**
Migration to 13.0

### CODE REVIEW

File: pos_order_remove_line/views/assets.xml [hunk 1]
```diff
@@ -7,9 +7,8 @@
     <template id="assets" inherit_id="point_of_sale.assets">
         <xpath expr="." position="inside">
             <link
+                href="/pos_order_remove_line/static/src/css/order_line.css"
```
Comment (ivantodorovich) [inline pos_order_remove_line/views/assets.xml]: Is there a reason to change `less` to `css`?

**Comment by misern2** [comment]
@JSeguiN97  Could you please attend the comments

**Comment by JSeguiN97** [comment]
@ivantodorovich I have changed the .css of the file to .less

**Review by ivantodorovich — APPROVED** [review]

**Review by misern2 — APPROVED** [review]

File: setup/.setuptools-odoo-make-default-ignore [hunk 2]
```diff
@@ -0,0 +1,2 @@
+# addons listed in this file are ignored by
+# setuptools-odoo-make-default (one addon per line)
```
Comment (chienandalu) [inline setup/.setuptools-odoo-make-default-ignore:2]: I don't think this file should belong in this PR

File: setup/README [hunk 3]
```diff
@@ -0,0 +1,2 @@
+To learn more about this directory, please visit
```
Comment (chienandalu) [inline setup/README:1]: Nor this one

**Review by chienandalu — COMMENTED** [review]
A little remark

**Pull request merged**